easybuild/framework/easyconfig/default.py

```diff
@@ -137,7 +137,7 @@
     'dependencies': [[], "List of dependencies", DEPENDENCIES],
     'hiddendependencies': [[], "List of dependencies available as hidden modules", DEPENDENCIES],
     'osdependencies': [[], "OS dependencies that should be present on the system", DEPENDENCIES],
-
+    'system_modules': [[], "System module dependencies that should be present on the system", DEPENDENCIES],
```

Comment:
i meant dependecy as in "without it, it won't work". dependencies in the form of external modules, like osdependencies but with modules.